browser/brave_stats/brave_stats_updater.cc

```diff
@@ -181,14 +185,16 @@ bool BraveStatsUpdater::MaybeDoThresholdPing(int score) {
   return false;
 }
 
-void BraveStatsUpdater::SetStatsUpdatedCallback(
+// static
+void BraveStatsUpdater::SetStatsUpdatedCallbackForTesting(
     StatsUpdatedCallback stats_updated_callback) {
-  stats_updated_callback_ = std::move(stats_updated_callback);
+  g_testing_stats_updated_callback = stats_updated_callback;
```

Comment:
I don't see us removing `stats_updated_callback_` and `stats_threshold_callback_` from the header file, unless I'm just missing it.